### PR TITLE
update favicons

### PR DESCRIPTION
**Pull request by Arhell**
https://github.com/kubeflow/website/issues/2976
after update
https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-2977--competent-brattain-de2d6d.netlify.app%2F#.YVHERrgzaMq

### CODE REVIEW

**Comment by Arhell** [comment]
/assign @Bobgy 
before https://realfavicongenerator.net/favicon_checker?protocol=https&site=www.kubeflow.org%2F#.YVJkybgzaMp
after https://realfavicongenerator.net/favicon_checker?protocol=https&site=deploy-preview-2977--competent-brattain-de2d6d.netlify.app%2F#.YVHERrgzaMq

**Comment by Arhell** [comment]
cc @RFMVasconcelos 
Check please, tnx
update here https://github.com/kubeflow/website/pull/2977#issuecomment-928522202

File: static/favicons/safari-pinned-tab.svg [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="310.000000pt" height="310.000000pt" viewBox="0 0 310.000000 310.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.14, written by Peter Selinger 2001-2017
```
Comment (Bobgy) [inline static/favicons/safari-pinned-tab.svg:8]: will this cause any legal issues?

Comment (theadactyl) [reply static/favicons/safari-pinned-tab.svg]: Hi Arhell, unfortunately we can't use this tool -- there is no OSS license for the source code, and potrace 1.14 uses the GPL license, which we have a policy against using in the project.

**Comment by Arhell** [comment]
I remove svg

**Pull request merged**